lib/poll.js

```diff
@@ -2,18 +2,35 @@
 const stats = require('./stats');
 const Logr = require('logr');
 const logrFlat = require('logr-flat');
-const log = Logr.createLogger({
-  type: 'flat',
-  reporters: {
-    flat: {
-      reporter: logrFlat,
-      options: {
-        timestamp: false,
-        appColor: true
+const logrSlack = require('logr-slack');
+
+let log;
+if (process.env.SLACK_HOOK) {
+  log = Logr.createLogger({
+    type: 'slack',
+    reporters: {
+      slack: {
+        reporter: logrSlack,
+        options: {
+          slackHook: process.env.SLACK_HOOK
+        }
       }
     }
-  }
-});
+  });
+} else {
+  log = Logr.createLogger({
+    type: 'flat',
+    reporters: {
+      flat: {
```

Comment:
we'll want flat all the time. we should have a reporters object and check env var and if so, add to reporters obj and then pass that to createLogger